WordPress/Classes/ViewRelated/Post/PostEditor+Publish.swift

```diff
@@ -95,6 +101,8 @@ extension PublishingEditor {
 
         mapUIContentToPostAndSave(immediate: true)
 
+        emitPostSaveEvent()
```

Comment:
I noticed that some lines below the publishing can be canceled, so maybe we should call this function once we are sure that the post will be saved, WDYT?

https://github.com/wordpress-mobile/WordPress-iOS/blob/d7584cea3350e88e1898b5b0e8f25d1462f4c95b/WordPress/Classes/ViewRelated/Post/PostEditor%2BPublish.swift#L106-L110

I potential place would be right before we track the `PostSave` event:
https://github.com/wordpress-mobile/WordPress-iOS/blob/d7584cea3350e88e1898b5b0e8f25d1462f4c95b/WordPress/Classes/ViewRelated/Post/PostEditor%2BPublish.swift#L151-L167

Reply:
That logic makes sense by me! Updated in https://github.com/wordpress-mobile/WordPress-iOS/pull/20522/commits/4e944ebdd90613e7a0960d37f64f5c4afe5bbb92.